README.md

```diff
@@ -1,36 +1,124 @@
+* [Contributing](#contributing)
+
+#### Introduction
+
+This repository demonstrates the ease of database migration from an on-premises SQL Server to an Amazon RDS instance.
+
+Deployment of this template will create two separate "environments". The first, an Amazon EC2 instance running an MS SQL 
+Server represents an on-premises environment and is therefore the *source* while the second environment, an Amazon RDS SQL Server instance is the *target* for the migration.
+
+The database is migrated using AWS DMS. It continuously synchronises changes in the "on-premises" instance with the cloud RDS instance.
+
+#### Summary
+
+This sample will deploy the two SQL Server instances (one EC2 and one Amazon RDS) in their own VPCs. 
+The SQL Server running in the EC2 instance represents the on-premises infrastructure and the Amazon RDS instance represents, 
+then create a sample database in both. No tables will be created at this point - creating of tables inside the database is left to the user. 
+A DMS migration task will also be created. Upon starting that task (eg; using console or aws cli), tables in the database will be continually replicated across from the EC2 instance to the Amazon RDS instance.
+
+### Architecture
+
+> diagram here
 
-## Requirements:
-- configured AWS CLI
-- configured AWS CLI profile
-- installed Python package manager PIP
-- S3 bucket in region where the solution will be deployed
+### Usage
 
-## Usage
-1. Get the solution by cloning this repository:
+#### Prerequisites
 
-`$ git clone https://github.com/aws-samples/aws-dms-sql-server.git`
+To deploy the application you will require an AWS account. If you don’t already have an AWS account, 
+create one at <https://aws.amazon.com> by following the on-screen instructions. 
+Your access to the AWS account must have IAM permissions to launch AWS CloudFormation templates that create IAM roles.
 
-2. Create S3 bucket: 
+#### Deployment
 
-`$ aws s3 mb s3://<BUCKET_NAME>/`
+The application is deployed as an [AWS CloudFormation](https://aws.amazon.com/cloudformation) template.
 
-3. Create a `.env` file and populate it with your own values:
+> **Note**  
+You are responsible for the cost of the AWS services used while running this sample deployment. There is no additional cost for using this sample. For full details, see the pricing pages for each AWS service you will be using in this sample. Prices are subject to change.
 
-`$ cp main/.env.example main/.env`
+1. Deploy the latest CloudFormation template by following the link below for your preferred AWS region:
 
-4. Run the deployment script: 
+|Region|Launch Template|
+|------|---------------|
+|**US East (N. Virginia)** (us-east-1) | [![Launch the EngagementMeter Stack with CloudFormation](docs/deploy-to-aws.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=aws-dms-sql-server&templateURL=https://s3.amazonaws.com/solution-builders-us-east-1/aws-dms-sql-server/v0.1/main.template)|
```

Comment:
```suggestion
|**US East (N. Virginia)** (us-east-1) | [![Launch the EngagementMeter Stack with CloudFormation](docs/deploy-to-aws.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=aws-dms-sql-server&templateURL=https://s3.amazonaws.com/solution-builders-us-east-1/aws-dms-sql-server/latest/main.template)|
```

Reply:
In this way we don't need to update the README every time we do a release

Reply:
Good catch. Fixed